tf-modules/ingest/variables.tf

```diff
@@ -267,12 +269,20 @@ variable "workflow_configurations" {
           keys(var.workflow_configurations.sf_event_sqs_to_db_records_types[workflow])
         ) :
         contains(["running", "completed", "failed"], status) &&
+        alltrue([
```

Comment:
What do you think about a test for this? e.g. `writeRecords() throws error and writes no records to PostgreSQL if 'execution' is not included`